### DISCUSSION TITLE
Jamulus won't start after upgrade to 3.10

### DISCUSSION THREAD

**Discussion by claaraghey** [jamulussoftware/jamulus#3186] [Technical Support & Help] [answered]
I'm on Windows 7 and have been using Jamulus faultlessly for years.

I installed the 3.10 upgrade, started Jamulus and got the following error:

The procedure entry point CreateDXGIFactory2 could not be located in the dynamic link library dxgi.dll

Any idea why this has happened and how I can fix it?

![error](https://github.com/jamulussoftware/jamulus/assets/146950847/dc0d1a08-61d7-4fca-b917-89054eaabe23)

**Comment by rdica** [comment, marked as answer]
From https://github.com/jamulussoftware/jamulus/blob/r3_10_0/ChangeLog:

Windows: Windows 64bit releases use Qt 6.5.2 now. Jamulus will from now on only support Windows 10 and above (#2300).

I'd suggest downgrading back to previous version, or upgrading the OS.

**Reply by claaraghey** [reply]
Would be nice to add a check to the installer for the OS version and warn people before proceeding.

**Reply by mcfnord** [reply]
You're the second person to fall into this trap that I'm aware of. I agree with you that an installer should just stop rather than do this. I'm not even sure we have a true installer, or just overwrite binaries.

**Comment by ann0see** [comment]
In theory the 32 Bit Version would still support old OS. But you should really update your PC as it's no longer supported by Microsoft.
